chapter_computational-performance/async-computation_vn.md

```diff
@@ -288,15 +288,22 @@ Given a finite amount of memory available on GPUs (and even on CPUs) this can le
 Some readers might have noticed that previous training routines made use of synchronization methods such as `item` or even `asnumpy`.
 -->
 
-*dịch đoạn phía trên*
+Hãy thử tưởng tượng trường hợp ta liên tục thêm các thao tác vào back-end bằng cách thực thi mã Python trên front-end.
+Ví dụ, trong một khoảng thời gian ngắn, front-end liên tục thêm vào một lượng lớn minibatch các tác vụ.
+Xét cho cùng, công việc trên có thể hoàn thành nhanh chóng nếu không có phép tính nào thực sự diễn ra trên Python.
+Nếu tất cả tác vụ trên có thể bắt đầu nhanh chóng trong cùng một lúc thì có thể dẫn đến dung lượng bộ nhớ sử dụng tăng đột ngột.
+Do dung lượng bộ nhớ có sẵn trên GPU (và ngay cả CPU) là có hạn, việc trên có thể gây ra sự tranh chấp tài nguyên hoặc thậm chí có thể gây hỏng chương trình.
+Vài bạn đọc có lẽ đã nhận ra rằng ở các quy trình huấn luyện trước, ta áp dụng phương pháp đồng bộ hoá như `item` hay ngay cả `asnumpy`.
 
 <!--
 We recommend to use these operations carefully, e.g., for each minibatch, such as to balance computational efficiency and memory footprint.
 To illustrate what happens let us implement a simple training loop for a deep network and measure its memory consumption and timing.
 Below is the mock data generator and deep network.
 -->
 
-*dịch đoạn phía trên*
+Chúng tôi khuyến nghị nên sử dụng các thao tác này một cách cẩn thận, ví dụ như đối với tứng minibatch, sao cho cân bằng giữa hiệu năng tính toán và vùng phủ bộ nhớ (*memory footprint*).
```

Comment:
```suggestion
Chúng tôi khuyến nghị nên sử dụng các thao tác này một cách cẩn thận cho từng minibatch, sao cho cân bằng giữa hiệu năng tính toán và mức chiếm dụng bộ nhớ (*memory footprint*).
```
mình nghĩ nên dịch đơn giản theo nghĩa `the amount of memory or disk space required by a program` thôi.